scripts/convert_audio_to_code.py

```diff
@@ -14,30 +17,30 @@
 # print("so both header files are output")
 # exit(1)
 
-py_file_abspath = os.path.abspath(__file__)
-
-os.chdir(os.path.join(os.path.dirname(py_file_abspath), ".."))
+if len(sys.argv) < 3:
```

Comment:
oof if we're going to use this python script, lets use argparse or something to get a nice cli  interface. My fault for not using it initially.